youtube_dl/extractor/howstuffworks.py

```diff
@@ -10,7 +10,7 @@
 
 
 class HowStuffWorksIE(InfoExtractor):
-    _VALID_URL = r'https?://[\da-z-]+\.howstuffworks\.com/(?:[^/]+/)*\d+-(?P<id>.+?)-video\.htm'
+    _VALID_URL = r'https?://[\da-z-]+\.howstuffworks\.com/(?:[^/]+/)*(\d+-)*(?P<id>.+?)-video\.htm'
```

Comment:
That's wrong by the way. It matches zero on more `\d+-` parts. Also do not capture group if you dont not use it.